IBAnimatable/MaskDesignable.swift

```diff
@@ -129,15 +135,52 @@ public extension MaskDesignable where Self: UIView {
     return path
   }
   
+  // MARK: - Parallelogram
+  // angle du triangle = 180 - topLeftAngle
+  // formule - 90
+  //  formule : offset = tan(angle) * hauteur
+  private func maskParallelogramFromString(mask:String) {
```

Comment:
I'm not sure that you need comments to explain this, but if comments, write them in english!